models/staging/stg_payments.sql

```diff
@@ -1,2 +1,2 @@
-{# select * from {{ source('stripe', 'payment') }} #}
-select * from raw.stripe.payment
\ No newline at end of file
+select * from {{ source('stripe', 'payment') }}
+{# select * from raw.stripe.payment #}
```

Comment:
We could remove the out-commented line here!

Reply:
done